modin/core/execution/ray/common/utils.py

```diff
@@ -102,6 +107,7 @@ def initialize_ray(
         )
 
         if cluster:
+            extra_init_kw["runtime_env"] = {"env_vars": env_vars}
```

Comment:
Since in case of cluster the Ray workers are already up and running, I am not sure this will have effect for Ray workers. That said, we could leave as it is.

cc @modin-project/modin-core, @modin-project/modin-ray

Reply:
@rkooo567, just to clarify. If we have an already set up Ray cluster with workers up and running, and then we call `ray.init("auto", "runtime_env"] = {"env_vars": env_vars})` to connect to the cluster, will Ray kill the previous workers and run new ones?

Reply:
It doesn't kill the previous one. It will just start a new one with new env vars. When the # of workers > num_cpus, the idle workers will be killed (so those workers "will" be killed eventually)